README.md

```diff
@@ -1 +1,8 @@
-# gold-project
\ No newline at end of file
+# gold-project
+
+Main Functionality
+1-[NinjaCourse] Add classes to a schedule to ensure no conflicts
+  See multiple variations to determine best schedule.
+
+Tasks
```

Comment:
May be weird to keep "Tasks" here. I'd remove Tasks from the README, unless it's supposed to serve some other purpose.